src/components/previewers/ImagePreviewer.jsx

```
@@ -49,108 +61,144 @@ const ImagePreviewer = ({
+                      description
+                    )}
+                  </div>
+                </div>
+                {!isEmoji && (
+                  <FaDownload
+                    className="w-5 h-5 cursor-pointer text-gray-700 hover:text-blue-500"
+                    onClick={handleDownload}
+                  />
+                )}
+              </div>
+              <div className="w-full h-[1px] bg-gray-200 my-2"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)